Problem2/Problem2.js

```diff
@@ -0,0 +1,8 @@
+function largestTwoNumbers(array){
+    if(array){
+        var Max = Math.max.apply(Math, array);
+        array.splice(array.indexOf(Max), 1);
+        var SecondMax = Math.max.apply(null, array);
```

Comment:
Am incercat sa gasesc o varianta pentru un singur loop insa fara rezultat. Cred ca o varianta cu un singur loop ar fi sortarea vectorului si afisarea ultimelor doua pozitii din vectorul sortat, doar ca cerinta problemei excludea sortarea vectorului.

Pentru prima intrebare, Math.max.apply(Math, array), aici trebuia sa fie tot null in loc de Math. Argumentul null trebuie sa existe deoarece rezultatul returnat in cazul Math.max.apply(array) ar fi -Infinity.